movement/monge_ampere.py

```diff
@@ -124,8 +125,8 @@ def __init__(self, mesh, monitor_function, **kwargs):
         :type rtol: :class:`float`
         :kwarg dtol: divergence tolerance for the residual
         :type dtol: :class:`float`
-        :kwarg fix_boundary_nodes: should all boundary nodes remain fixed?
-        :type fix_boundary_nodes: :class:`bool`
+        :kwarg fixed_boundary_segments: boundary segments to hold fixed
```

Comment:
It would be clearer if this was a bit more precise. Something like "integer markers/labels corresponding to boundary segments...". Also would be good to explicitly mention the `"on_boundary"` option here.

Reply:
Good idea. Fixed in e5b0519afb3bd3640797f2cfec486cc54cfc106e.